### ISSUE TITLE
Range of bows is to low compared to slings

### ISSUE THREAD

**Issue by BjoHart** [CleverRaven/Cataclysm-DDA#60916]
### Is your feature request related to a problem? Please describe.

I am an archer in real life.
It frustrates me that the range of the bows in the game is very short, especially compared to slingshots.

### Solution you would like.

The range of the bows should be adjusted upwards

### Describe alternatives you have considered.

Or maybe change the range of the slingshots downwards

**Comment by Zireael07** [comment]
The same could be said of guns - IIRC all the ranges are "compressed" to fit in the reality bubble.

**Comment by BjoHart** [comment]
This is true, but that the slingshot can reach range 20 and the longbow only 18 is not very realistic in terms of ratio

**Comment by Aerin-of-the-Toast** [comment]
This is mostly an error with throwing, not archery. All ranged weapons except throwing weapons are subject to the same amount of "range compression," and throwing only gets a pass because if it was also as compressed you'd barely be able to throw anything farther than like 3 tiles, which would make stuff like grenades completely unusable. 

Range compression will likely get relaxed a bit at some point if the game gets optimized enough that the reality bubble can be expanded.

**Comment by Alm999** [comment]
But the "staff sling" uses the f[ire] mode, not the t[hrow] one. For all intents and purposes it acts as a firearm.

Moreover, "staff sling" has a dispersion of just 200, while its competition (in primitiveness) from the archery branch (short/long bow) has a dispersion of 1000! A case of laser-guided rocks, if I may say so.
